parsers/eiffel.c

```diff
@@ -149,6 +149,7 @@ static const keywordTable EiffelKeywordTable [] = {
 	{ "indexing",       KEYWORD_indexing   },
 	{ "infix",          KEYWORD_infix      },
 	{ "inherit",        KEYWORD_inherit    },
+	{ "insert",         KEYWORD_insert     },
```

Comment:
Updated thanks.

I was considering capturing redefinitions and the difference between inherit and insert but the distinction is not necessary now.